safekeeper/src/pull_timeline.rs

```diff
@@ -94,33 +94,31 @@ async fn pull_timeline(status: TimelineStatus, host: String) -> Result<Response>
         status.acceptor_state.epoch
     );
 
-    let conf = &GlobalTimelines::get_global_config();
+    let tli = match GlobalTimelines::get(ttid) {
+        Ok(tli) => tli,
+        Err(_) => {
+            bail!("Expected to fetch single timeline, got no timelines");
+        }
+    };
 
-    let client = reqwest::Client::new();
-    // TODO: don't use debug dump, it should be used only in tests.
-    //      This is a proof of concept, we should figure out a way
-    //      to use scp without implementing it manually.
-
-    // Implementing our own scp over HTTP.
-    // At first, we need to fetch list of files from safekeeper.
-    let dump: debug_dump::Response = client
```

Comment:
We want to keep this logic for making debug_dump requests on another safekeeper, see previous comment about Deserialize.